### PR TITLE
Create initila BFF HTTP Server to accept diffent type of requests 

### PR DESCRIPTION
**Pull request by yangli2014**
Create the initial BFF Akka HTTP server to accept GET, POST, and DELETE operation

### CODE REVIEW

File: bff/src/main/java/org/opennms/devjam2022/bff/BFFApplication.java [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+    final Http http = Http.get(actorSystem);
+
+    BFFApplication app = new BFFApplication();
+
+    final CompletionStage<ServerBinding> binding = http.newServerAt("localhost", 8081)
+        .bind(app.createRoute());
+
+    System.out.println("Server online at http://localhost:8081/\nPress RETURN to stop...");
+    System.in.read();
```
Comment (denkonakov) [inline bff/src/main/java/org/opennms/devjam2022/bff/BFFApplication.java:59]: Should be OK for now, but I would try to eliminate this one ASAP and implement something more proper

File: shared_lib/src/main/java/org/opennms/devjam2022/lib/User.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+package org.opennms.devjam2022.lib;
+
+public class User {
+  private long id;
+  private String firstName;
+  private String lastName;
+  private Address address;
+
+  public User(long id, String firstName, String lastName, Address address) {
```
Comment (denkonakov) [inline shared_lib/src/main/java/org/opennms/devjam2022/lib/User.java:37]: Can we use Lombook just to avoid of this boilerplate code?

**Review by denkonakov — APPROVED** [review]
LGTM for our "demo" purposes )

File: shared_lib/src/main/java/org/opennms/devjam2022/lib/User.java (hunk 2 above)
Comment (yangli2014) [reply shared_lib/src/main/java/org/opennms/devjam2022/lib/User.java:37]: sure. But since we already use your branch. we don't need the share_lib for now.

**Pull request merged**